core/services/relay/evm/binding/lookup.go

```diff
@@ -0,0 +1,79 @@
+package binding
+
+import (
+	"sync"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/types"
+)
+
+type readValues struct {
+	address  string
+	contract string
+	method   string
+}
+
+type lookup struct {
+	mu              sync.RWMutex
+	contractMethods map[string][]string
```

Comment:
This whole file uses method terminology which we should avoid because of evm association, is readName okay?